internal/action/workspace.go

```diff
@@ -245,6 +247,20 @@ func NewWorkspaceConfig(ctx context.Context, client *tfe.Client, workspaces []*W
 	return module, nil
 }
 
+// WriteModuleFile is a simple utility to marshal the passed module and write it to the passed file path
+func WriteModuleFile(module *tfconfig.Module, filePath string) error {
+	b, err := json.MarshalIndent(module, "", "\t")
```

Comment:
```suggestion
	b, err := json.MarshalIndent(module, "", "  ")
```

Not that it really matters

Reply:
in this house, tabs are only for make files